src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfUserIpv4MapCache/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,571 @@
+expected_output = {
+    "vrf": "User",
+    "iid": 4100,
+    "number_of_entries": 2186,
+    "eid": {
+        "0.0.0.0/0": {
+            "subnet": "/0",
+            "uptime": "1w6d",
+            "expire": "never",
+            "source": [
+                "static-send-map-request"
+            ],
+            "rloc": {
+                "status": "negative_cache_entry",
```

Comment:
this is value. so we can take as is.
```suggestion
                "status": "Negative cache entry",
```

Reply:
Updated